content/blog/performant-elm/index.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Performant Elm, Part I
+date: "2019-07-08T12:00:00.000Z"
+description: A tale of performance optimization in Elm.
+---
+
+You have written your first application in Elm. Congratulations my friend,
+the hard work has finally paid off. Now it's time to relax and enjoy the
+pure bliss that is maintaining and refactoring Elm code.
+
+But wait a second, someone tells you the app feels a bit sluggish to use.
+Mh. I thought that writing pure functions would automatically make code
+fast. Okay. Let's try to stay calm.
+
+The app looks fast at a first glance. To be fair, you're probably using a
+quad-core machine with 16 GB of RAM. Maybe you should see how the app
+behaves on a lower end device. In Chrome, you can bring up the inspector,
+then go to Performance tab and choose how much you want to throttle your
+CPU.
```

Comment:
`In Chrome, you can bring up the inspector...` -> adding "can" here feels more expository than narrative which is the writing style of the rest of the post. Maybe remove "can"? `In Chrome, you bring up the inspector...`